drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/MetaDataColumnPage.java

```diff
@@ -19,28 +19,29 @@
 import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
 
-import com.google.gwt.user.client.ui.Widget;
+import com.google.gwt.regexp.shared.RegExp;
 import org.drools.workbench.screens.guided.dtable.client.resources.i18n.GuidedDecisionTableErraiConstants;
 import org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.BaseDecisionTableColumnPage;
 import org.drools.workbench.screens.guided.dtable.client.wizard.column.plugins.MetaDataColumnPlugin;
 import org.jboss.errai.ui.client.local.spi.TranslationService;
 import org.uberfire.client.callbacks.Callback;
 import org.uberfire.client.mvp.UberElement;
-import org.uberfire.client.mvp.UberView;
 
 import static org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.DecisionTableColumnViewUtils.nil;
 
 @Dependent
 public class MetaDataColumnPage extends BaseDecisionTableColumnPage<MetaDataColumnPlugin> {
 
     private View view;
+    private RegExp containWhiteSpacePattern;
 
     @Inject
     public MetaDataColumnPage(final View view,
                               final TranslationService translationService) {
         super(translationService);
 
         this.view = view;
+        this.containWhiteSpacePattern = RegExp.compile("\\s");
```

Comment:
@manstis @jomarko @jhrcek Are you speaking about the key or the value of the metadata? In the first case it has to be a valid Java identifier while in the second there are no restrictions (I just checked that also the space works).